apps/super-app/src/App.test.tsx

```diff
@@ -3,27 +3,30 @@ import { render, screen } from '@testing-library/react';
 import App from './App';
 import { MemoryRouter } from 'react-router-dom';
 
-test('renders a nonsense homepage', () => {
+test('renders LandingPage component on load homepage', () => {
   render(
     <MemoryRouter>
       <App />
     </MemoryRouter>
   );
-  const todoLink = screen.getByText(/TODO/i);
-  expect(todoLink).toBeInTheDocument();
+  const header = screen.getByText('Free Support Services');
+  expect(header).toBeInTheDocument();
+  const twc2 = screen.getByText('Transient Workers Count Too');
+  expect(twc2).toBeInTheDocument();
+  const pbsg = screen.getByText('Pro Bono SG');
+  expect(pbsg).toBeInTheDocument();
 });
 
-test('renders some nonsense buttons on /scratch', () => {
+test('make sure app initializes on /landing-page', () => {
```

Comment:
The pre-established style is to use the singular present tense as if the codebase is doing something, e.g. `(it) renders some buttons`, `(it) should do X`. I think it might have been part of [mocha](https://mochajs.org/#assertions)'s legacy, but I've seen the `should do X when Y` pattern in quite a few places tbh.